config/site_settings.yml

```diff
@@ -2931,9 +2931,6 @@ developer:
     client: true
     default: 25000
     max: 99000
-  flush_timings_secs:
```

Comment:
I don't know who might have update this but we should also add a migration to drop any overrides for this site setting from the database.